lib/watchaggregator/aggregator.go

```diff
@@ -458,16 +466,16 @@ func updateInProgressSet(event events.Event, inProgress map[string]struct{}) {
 // air, they were just considered as a sensible compromise between fast updates and
 // saving resources. For short delays the timeout is 6 times the delay, capped at 1
 // minute. For delays longer than 1 minute, the delay and timeout are equal.
-func notifyTimeout(eventDelayS int) time.Duration {
-	shortDelayS := 10
+func notifyTimeout(eventDelayMs int) time.Duration {
```

Comment:
This thing should take a `time.Duration` to begin with to reduce the unit conversion gymnastics. Then we get a common `a.notifyTimeout = notifyTimeout(a.notifyDelay)` after the if-clauses above.